### DISCUSSION TITLE
@nuxt/content V2 renders <a> tag inside heading tags (h1, h2 ...)

### DISCUSSION THREAD

**Discussion by benoitdemaegdt** [nuxt/content#1248] [Q&A] [answered]
Hey there 👋 

I am using @nuxt/content along with [@tailwind/typography](https://tailwindcss.com/docs/typography-plugin).
This works well, I am able to get tailwind style this way 👇 

```
<div class="mt-6 prose prose-emerald prose-lg text-gray-500 mx-auto">
  <ContentRenderer :value="data" />
</div>
```

However, I'am annoyed by @nuxt/content rendering a `<a>` tag inside all heading tags.
Meaning that this markdown content : 
```
### Title
This is my markdown content
```
Will render like this : 
```
<h2 id="Title">
  <a aria-current="page" href="#title" class="router-link-active router-link-exact-active"><!--[-->title<!--]-->
    Title
  </a>
</h2>
<p>
  This is my markdown content
</p>
```

Then Tailwind will apply `<a>` tag style and the title now looks like an external link.

Does anyone knows if there is a way **not** to render this `<a>` tag inside heading tags ? 
Meaning that previous markdown content would render like this :
```
<h2 id="Title">Title</h2>
<p>This is my markdown content</p>
```

**Comment by nozomuikuta** [comment, marked as answer]
@benoitdemaegdt 

You can override how Nuxt Content renders HTML elements in markdown.
Please refer to [a section about Prose](https://content.nuxtjs.org/guide/writing/markdown#prose).
It's also worth checking [the list of Prose components](https://content.nuxtjs.org/api/components/prose).

**Reply by benoitdemaegdt** [reply]
Thank you so much @NozomuIkuta, I wasn't aware of this way of overriding Prose components. It perfectly works 👌
